features/service-operation.feature

```diff
@@ -0,0 +1,36 @@
+Feature: Command output from service operations like stop/start/restart
+  service-manager should return the correct exit code from stop/start/restart command
+
+  @operation
+  Scenario Outline: Boot and execute service operations like stop/start/restart
+    Given box is <box>
+    And provider is <provider>
+    And a file named "Vagrantfile" with:
+    """
+    require 'vagrant-libvirt'
+
+    Vagrant.configure('2') do |config|
+      config.vm.box = '<box>'
+      config.vm.box_url = 'file://../boxes/<box>-<provider>.box'
+      config.vm.network :private_network, ip: '<ip>'
+      config.vm.synced_folder '.', '/vagrant', disabled: true
+      config.servicemanager.services = 'docker'
+    end
+    """
+
+    When I successfully run `bundle exec vagrant up --provider <provider>`
+      And I successfully run `bundle exec vagrant service-manager stop docker`
```

Comment:
Even we cannot test `restart` command due to [this issue](https://github.com/projectatomic/adb-atomic-developer-bundle/issues/342). However, to test this command, probably one of provider could be used. Instead of checking all.